projects/Mallard/src/helpers/files.ts

```diff
@@ -12,25 +12,14 @@ import { withCache } from './fetch/cache'
 import { updateListeners } from 'src/download-edition/download-and-unzip'
 import { getSelectedEditionSlug } from 'src/hooks/use-edition-provider'
 import { editionsListCache } from 'src/helpers/storage'
-interface BasicFile {
-    filename: string
-    path: string
-    size: number
-    id: string
-}
-interface OtherFile extends BasicFile {
-    type: 'other' | 'archive' | 'json'
-}
-interface IssueFile extends BasicFile {
-    issue: Issue
-    type: 'issue'
-}
 
-const regex = /\d{4}-\d{2}-\d{2}/gm // this matches issue date, i.g. 2020-02-01
+const ISSUE_DATE_REGEX = /\d{4}-\d{2}-\d{2}/gm
```

Comment:
we can keep the comment that might be helpful for someone new to the codebase

Reply:
Yes, much clearer. I have restored the comment